espnet2/asr/transducer/beam_search_transducer.py

```diff
@@ -244,18 +261,58 @@ def default_beam_search(self, enc_out: torch.Tensor) -> List[Hypothesis]:
             nbest_hyps: N-best hypothesis.
 
         """
+        logging.info("decoder input length: " + str(enc_out.shape[0]))
```

Comment:
> Do you mean encoder input length?

It's the encoder's output length, which is equal to decoder input length? I simply used the logging format from (here)[https://github.com/espnet/espnet/blob/master/espnet/nets/beam_search.py#L362].
> Also, same comment as before, we could print encoder length before calling beam search.

Yes, if we do logging in `asr_inference.py`, then we can remove this line from here.

Reply:
> It's the encoder's output length, which is equal to decoder input length? I simply used the logging format from [here](https://github.com/espnet/espnet/blob/master/espnet/nets/beam_search.py#L362).

I understood but it may be a bit confusing because we deal with two axes for the Transducer. The total steps with label expansion will be more than *T* (`T + u_max`, `T * nsteps`, `T * adaptive_nsteps`, etc).
I don't mind this information thought.

Reply:
When we finalize the logging part, we can add this type of detailed information. I think it'll be really helpful.